asminc/apple2.inc

```diff
@@ -44,6 +44,9 @@ SET80COL:=      $C001   ; Enable 80 column store
 RD80COL :=      $C018   ; >127 if 80 column store enabled
 RD80VID :=      $C01F   ; >127 if 80 column video enabled
 
+VIDEOMODE_40x24 = $15
+VIDEOMODE_80x24 = $00
```

Comment:
It somehow feels to me that this doesn't belong here. This file is supposed to be "universal Apple II truth", not about cc65 specific things.

Reply:
Understood.
I'll move them away to videomode.s for now; but on that subject, I often miss having asm defines in my programs for values (like CH_CURS_*, APPLE_IIE, etc). Should I introduce a file specific to those (in another PR)?